docs/contributing.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Contributing
+nav_order: 11
+---
+
+# How to contribute
+
+There are many ways you can contribute to add support for your language or improve translation quality!
+
+Don't hesitate to ask questions in the Github issues or on Matrix [#firefoxtranslations:mozilla.org](https://matrix.to/#/#firefoxtranslations:mozilla.org). 
+
+## Native speakers
+
+### Providing feedback
+
+Feedback from native speakers of the languages we already support is very valuable to us. 
+
+If you are a native speaker of a language we translate to and know the source language to some extent, you can help us understand the issues with the trained models.
+Please test web-page translation functionality in Firefox Nightly on your favourite websites and report any issues you see by creating one Github issue per target language.
+We will label it later as ["feedback"](https://github.com/mozilla/translations/issues?q=is%3Aissue+is%3Aopen+label%3Afeedback+) and discuss whether a fix can be implemented. Here is a [good example](https://github.com/mozilla/translations/issues/816) of a contributor reporting an issue.
+Feel free to do it in a format convenient to you.
+
+We are currently working on better feedback mechanisms, integrated in Firefox and as a standalone contribution platform.
+
+### Finding new datasets
+
+You can look at the [past training configs](https://github.com/mozilla/translations/tree/main/configs) and play with the [config-generator](https://github.com/mozilla/translations/blob/main/utils/config_generator.py) to see what open-source datasets are currently available for training.
+If you know some datasets that are not in the configs, please create an issue or consider mentioning it in the existing issues labeled as ["data sources"](https://github.com/mozilla/translations/labels/data%20sources).
+
+We need both parallel translation corpus and high-quality monolingual datasets, especially for lower resource languages.
+
+See also [Data importers docs]([data.md](data.md)).
+
+### Contributing datasets
+
+We use [OPUS](https://opus.nlpl.eu/) as the main resource for parallel translation datasets. If you know of a dataset that is not on OPUS, please create an issue in 
+https://github.com/Helsinki-NLP/OPUS-ingest/issues.
+
+Another way is to submit your dataset to Hugging Face and let us know that we should use it. 
+
+### Inspecting datasets
+
+Another way to contribute is to look at specific parallel datasets for your language pair.
+
+The main tool for this is [OpusCleaner](https://github.com/hplt-project/OpusCleaner).
+If the dataset looks too noisy, create a PR with exclusion rules in the [config-generator](https://github.com/mozilla/translations/blob/main/utils/config_generator.py).
```

Comment:
Do we have examples of exclusion rules?

Reply:
Yes, `skip_datasets` is in the beginning of the [file](https://github.com/mozilla/translations/blob/2fe2ada160751495fa8d60226f53607ea457f331/utils/config_generator.py#L35), it's not per language though. I can clarify that.